routers/install/install.go

```diff
@@ -421,9 +422,10 @@ func SubmitInstall(ctx *context.Context) {
 		cfg.Section("server").Key("LFS_START_SERVER").SetValue("false")
 	}
 
-	if len(strings.TrimSpace(form.SMTPHost)) > 0 {
+	if len(strings.TrimSpace(form.SMTPAddr)) > 0 {
 		cfg.Section("mailer").Key("ENABLED").SetValue("true")
-		cfg.Section("mailer").Key("HOST").SetValue(form.SMTPHost)
+		cfg.Section("mailer").Key("SMTP_ADDR").SetValue(form.SMTPAddr)
+		cfg.Section("mailer").Key("SMTP_PORT").SetValue(form.SMTPPort)
```

Comment:
Should the install form be updated accordingly?

https://github.com/clarfonthey/gitea/blob/mailer-settings/templates/install.tmpl

Reply:
I thought I did, but I can double check

Reply:
I fixed it in https://github.com/go-gitea/gitea/pull/18982/commits/8d207029e91379927058c040b59bc53ffa34f518